src/common/hotbar-store/add-to-active-hotbar.injectable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { IComputedValue } from "mobx";
+import type { CatalogEntity } from "../catalog";
+import { bind } from "../utils";
+import activeHotbarInjectable from "./active-hotbar.injectable";
+import type { Hotbar } from "./hotbar";
+
+interface Dependencies {
+  hotbar: IComputedValue<Hotbar>;
+}
+
+function addToActiveHotbar({ hotbar }: Dependencies, entity: CatalogEntity, cellIndex?: number) {
```

Comment:
Could be lambda. Superficial.

Reply:
I prefer the `bind` pattern instead as I find it easier to read.